lib/caesar_cypher.rb

```diff
@@ -1,2 +1,40 @@
 class CaesarCypher
+  LOWER_A_ORD = "a".ord
+  UPPER_A_ORD = "A".ord
+
+  def initialize(shift)
+    @shift = shift
+  end
+
+  def encrypt(input)
+    input.gsub(/[a-zA-Z]/) do |s|
+      if s >= "a"
```

Comment:
Don't love having a conditional here.  What else can be done for naming or logic?